src/results/results.jl

```diff
@@ -183,6 +183,11 @@ function combine_results(p::REoptInputs, bau::Dict, opt::Dict, bau_scenario::BAU
         ("ElectricTariff", "year_one_export_benefit_after_tax"),
         ("ElectricTariff", "year_one_coincident_peak_cost_before_tax"),
         ("ElectricTariff", "lifecycle_coincident_peak_cost_after_tax"),
+        ("ElectricTariff", "annual_electric_gross_purchase_cost_series"),
```

Comment:
@rathod-b don't we have to add any/all of the results that you added for ElectricTariff and ElectricLoad (except for ones that involve battery) to the bau results?